tests/dbaas_policy.robot

```diff
@@ -7,27 +7,201 @@ Resource            ../resources/keywords/dbaas_policy.resource
 
 Suite Setup         Configure DBaaS Policy Namespaces
 Suite Teardown      Tear Down The Test Suite
-Test Teardown       Close Browser
+Test Teardown       Tear Down DBaaSPolicy Test Case
 Force Tags          UI  DBaaS
 
 *** Test Cases ***
-Scenario: Verify Invalid Namespace Error Message For Project Admin To Create DBaaS Policy
+Scenario: Verify Invalid Namespace Error Message For Project Admin To Create DBaaS Policy on Admin View
     [Tags]  dbaas-test-01
     Given The User Logs In To Openshift As OCP_LDAP_PROJECT_ADM
     When User Filters Project ${DBaaSTestNS} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     Then Import Provider Account Failure For DBaaS Policy
 
-Scenario: Verify Invalid Namespace Error Message For Service Admin To Create DBaaS Policy
+Scenario: Verify Invalid Namespace Error Message For Service Admin To Create DBaaS Policy on Admin View
     [Tags]  dbaas-test-02
     Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
     When User Filters Project ${DBaaSTestNS} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     Then Import Provider Account Failure For DBaaS Policy
 
-Scenario: Verify Invalid Namespace Error Message For Developer To Create DBaaS Policy
+Scenario: Verify Invalid Namespace Error Message For Developer To Create DBaaS Policy on Admin View
     [Tags]  dbaas-test-03
     Given The User Logs In To Openshift As OCP_LDAP_DEV
     When User Filters Project ${DBaaSTestNS} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     Then Import Provider Account Failure For DBaaS Policy
+
+Scenario: Verify Invalid Namespace Error Message For Project Admin To Create DBaaS Policy on Dev View
+    [Tags]  dbaas-test-04
+    Given The User Logs In To Openshift As OCP_LDAP_PROJECT_ADM
+    When User Navigates To Import MongoDB Provider Account On Developer View
+    Then Import Provider Account Failure For DBaaS Policy
+
+Scenario: Verify Invalid Namespace Error Message For Service Admin To Create DBaaS Policy on Dev View
+    [Tags]  dbaas-test-05
+    Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
+    When User Navigates To Import MongoDB Provider Account On Developer View
+    Then Import Provider Account Failure For DBaaS Policy
+
+Scenario: Verify Invalid Namespace Error Message For Developer To Create DBaaS Policy on Dev View
+    [Tags]  dbaas-test-06
+    Given The User Logs In To Openshift As OCP_LDAP_DEV
+    When User Navigates To Add Topology Screen From Developer View      MongoDB     ${DBaaSTestNS}     True
```

Comment:
How does this keyword works with the MongoDB and ${DBaaSTestNS}?

Reply:
User Navigates To Add Topology Screen From Developer View keyword accepts three parameters,
Database provider, Namespaces and indicator for DBaaSPersona is developer
We directly added the values for first and last, where as the third one retrieved from login.resource which would be imported with setup steps.

Reply:
OK nice